src/System.Management.Automation/engine/Modules/AnalysisCache.cs

```diff
@@ -103,6 +103,12 @@ internal class AnalysisCache
                 var moduleManifestProperties = PsUtils.GetModuleManifestProperties(modulePath, PsUtils.FastModuleManifestAnalysisPropertyNames);
                 if (moduleManifestProperties != null)
                 {
+                    if (!Configuration.PowerShellConfig.Instance.IsImplicitWinCompatEnabled() && ModuleIsEditionIncompatible(modulePath, moduleManifestProperties))
```

Comment:
Why not GetPolicySetting()?
For enterprise adoption GPO would be better.

Reply:
Good point. We've discussed this with @daxian-dbw and @joeyaiello and the decision was to leave this setting as non-GPO, with the reason being that it does Not control the security boundary (because with any value of this setting users can still use the WinCompat feature explicitly using the cmdlet parameters).
Besides, machine-wide configs can be pushed using GPO anyways.